stable/appmesh-prometheus/README.md

```diff
@@ -21,6 +21,29 @@ helm upgrade -i appmesh-prometheus eks/appmesh-prometheus \
 --namespace appmesh-system
 ```
 
+Enable EBS CSI Driver:  
+- Initialize iam-oidc-provider
+```
+eksctl utils associate-iam-oidc-provider --region=us-west-2 --cluster=appmeshtest --approve
+```
+- Create IAM role
+```
+eksctl create iamserviceaccount \
+  --name ebs-csi-controller-sa \
+  --namespace kube-system \
+  --cluster appmeshtest \
```

Comment:
The customer would need to replace cluster name with their cluster name right?

Reply:
Thank you, I re-corrected with a replaceable label here for cluster name